### DISCUSSION TITLE
Is there a way to mute just one tab?

### DISCUSSION THREAD

**Discussion by danielfalbo** [qutebrowser/qutebrowser#5840] [Q&A] [answered]
Is there a way to mute just one tab, like in Safari?

https://support.apple.com/guide/safari/mute-audio-in-tabs-ibrwae910b47/mac

**Comment by toofar** [comment, marked as answer]
`:tab-mute`, bound to Alt+m by default, toggles the muted/audible status. It should put a `[M]` in the tab title too assuming `{audio}` is in the tab title format setting.

**Reply by danielfalbo** [reply]
Thank you! 🌟
